controllers/articles.controllers.js

```diff
@@ -0,0 +1,25 @@
+const {
+  fetchArticleById,
+  fetchAllArticles,
+} = require("../models/articles.models");
+
+exports.getArticleById = (req, res, next) => {
+  const { article_id } = req.params;
+  fetchArticleById(article_id)
+    .then((article) => {
+      res.status(200).send({ article });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
+
+exports.getArticles = (req, res, next) => {
+  fetchAllArticles()
+    .then((articles) => {
+      res.status(200).send({ articles });
+    })
+    .catch((err) => {
+      next(err);
+    });
+};
```

Comment:
I like the fact that you've created a different files for different controllers. 🙂